src/main/java/org/bytecodeparser/instruction/PrettyPrintTypes.java

```diff
@@ -0,0 +1,42 @@
+package org.bytecodeparser.instruction;
+
+import org.bytecodeparser.utility.ThrowingFunction;
+
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.util.function.Function;
+
+public enum PrettyPrintTypes {
+    SIMPLE(dataInputStream -> InstructionArguments.builder().build(), arguments -> ""),
```

Comment:
name 'SIMPLE' is not informative, you can use EMPTY

Reply:
fixed